packages/webdriverio/package.json

```diff
@@ -70,7 +70,7 @@
     "ts-node": "^10.9.1",
     "tsup": "^6.7.0",
     "typescript": "^4.8.4",
-    "webdriverio": "^8.8.2"
+    "webdriverio": "8.8.2"
```

Comment:
Latest webdriverio breaks our typescript build due to this bug with types https://github.com/webdriverio/webdriverio/issues/11055. Will open a tech debt ticket to fix it.

Reply:
https://github.com/dequelabs/axe-core-npm/issues/829